esp-bootloader-esp-idf/CHANGELOG.md

```diff
@@ -9,22 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `OtaUpdater::next_partition` (#4339)
 
 ### Changed
 
+- `OtaUpdater::with_ota` is now public (#4413)
```

Comment:
I think this should be logged as a pair of Removed-Added changelog entries. `with_ota` simply doesn't exist anymore :)

Reply:
Done!